doc/source/rllib/rllib-concepts.rst

```diff
@@ -230,76 +230,60 @@ First, check out the `PPO trainer definition <https://github.com/ray-project/ray
 
```

Comment:
Unrelated: rllib-concepts.rst is not a very good name for a file that describes "How To Customize Policies".

Reply:
ikr